.travis.yml

```diff
@@ -38,11 +37,24 @@ jobs:
         - PYTHON_EXECUTABLE="/usr/local/bin/python3"
       script: sh .travis/run_tests.sh
     - stage: "Deploy"
-      name: "conda"
-      if: branch = master AND NOT type = pull_request
+      name: "conda linux"
+      #if: branch = master AND NOT type = pull_request
```

Comment:
@OwenArnold Could you place this check back in. This has been publishing additional Linux packages that aren't needed.

Reply:
Agreed.